src/xcode/ENA/ENA/Source/Services/HealthCertificate/Models/Extensions/RecoveryEntry+Extension.swift

```diff
@@ -15,6 +15,10 @@ extension RecoveryEntry {
 		return ISO8601DateFormatter.justLocalDateFormatter.date(from: certificateValidUntil)
```

Comment:
Can 'localCertificateValidityStartDate' and 'localCertificateValidityEndDate' be removed from the extension?